src/main/java/training/TrainingRobotManager.java

```diff
@@ -1,19 +1,26 @@
 package training;
 
+import org.littletonrobotics.junction.AutoLog;
+import training.commands.MoveAngularMotorByVoltage;
+import training.commands.MoveLinearMotorByVoltage;
 import utils.DefaultRobotManager;
+import utils.joysticks.SmartJoystick;
 
 public class TrainingRobotManager extends DefaultRobotManager {
 
     private Robot robot;
+    private SmartJoystick joystick;
 
     @Override
     public void trainingInit() {
-        this.robot = new Robot();
+        robot = new Robot();
+        joystick = new SmartJoystick(RobotConstants.ID_OF_FIRST_JOYSTICK);
     }
 
     @Override
     public void trainingPeriodic() {
         // add stuff...
+        joystick.A.whileTrue(new MoveAngularMotorByVoltage(1));
```

Comment:
the training periodic happens every 20 miliseconds, you only need to configurate the joystick button once so you can easily put this in the trainingInit